compiler/x/env/OMRCPU.cpp

```diff
@@ -254,9 +279,62 @@ OMR::X86::CPU::is(OMRProcessorArchitecture p)
    return _processorDescription.processor == p;
    }
 
+bool
+OMR::X86::CPU::is_feature_disabled(uint32_t feature)
```

Comment:
1.  I'm not sure why snake case got introduced in this class for naming functions, but please use camel case for consistency with the rest of the codebase.

2.  I think this would be better named `isFeatureDisabledByOption` to make it clear that is the reason the feature is disabled.  Earlier in this PR there are features disabled because the OS doesn't support it, so naming it this way provides more clarity.